plugins/image.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: image
+spec:
+  version: v1.0.10
+  homepage: https://github.com/pete911/kubectl-image
+  platforms:
+    - selector:
+        matchLabels:
+          os: darwin
+          arch: amd64
+      uri: https://github.com/pete911/kubectl-image/releases/download/v1.0.10/kubectl-image_1.0.10_darwin_amd64.tar.gz
+      sha256: 3fc45027070ae0a5f60a60a57bcb2d45e96a65a197fad5c119e2841ce50d5e20
+      bin: kubectl-image
+      files:
+        - from: kubectl-image
+          to: .
+        - from: LICENSE
+          to: .
+    - selector:
+        matchLabels:
+          os: darwin
+          arch: arm64
+      uri: https://github.com/pete911/kubectl-image/releases/download/v1.0.10/kubectl-image_1.0.10_darwin_arm64.tar.gz
+      sha256: afbbb90c5b98233912eb18baa282b4bca8b7c19f72adb0667db5256d95e2fba4
+      bin: kubectl-image
+      files:
+        - from: kubectl-image
+          to: .
+        - from: LICENSE
+          to: .
+    - selector:
+        matchLabels:
+          os: linux
+          arch: amd64
+      uri: https://github.com/pete911/kubectl-image/releases/download/v1.0.10/kubectl-image_1.0.10_linux_amd64.tar.gz
+      sha256: 13f24ec268f4249b29ce4fc5ce74b4c7d73e6607480467f74c2ed89962930c91
+      bin: kubectl-image
+      files:
+        - from: kubectl-image
+          to: .
+        - from: LICENSE
+          to: .
+    - selector:
+        matchLabels:
+          os: linux
+          arch: arm64
+      uri: https://github.com/pete911/kubectl-image/releases/download/v1.0.10/kubectl-image_1.0.10_linux_arm64.tar.gz
+      sha256: ee3fe615c78de5394983f68f2fbb58ac1f707e8c173002a798ef32a8b9dbd372
+      bin: kubectl-image
+      files:
+        - from: kubectl-image
+          to: .
+        - from: LICENSE
+          to: .
+    - selector:
+        matchLabels:
+          os: windows
+          arch: amd64
+      uri: https://github.com/pete911/kubectl-image/releases/download/v1.0.10/kubectl-image_1.0.10_windows_amd64.zip
+      sha256: 89425211e89b7bb144aa7590f3aad35cdecf9dfb58bf3281d0912d4789cebacb
+      bin: kubectl-image.exe
+      files:
+        - from: kubectl-image.exe
+          to: .
+        - from: LICENSE
+          to: .
+    - selector:
+        matchLabels:
+          os: windows
+          arch: arm64
+      uri: https://github.com/pete911/kubectl-image/releases/download/v1.0.10/kubectl-image_1.0.10_windows_arm64.zip
+      sha256: 241593262a759c71c8d1dd8c0a4aa42277a60ab07846f515d2ebcbbc057a94a4
+      bin: kubectl-image.exe
+      files:
+        - from: kubectl-image.exe
+          to: .
+        - from: LICENSE
+          to: .
+  shortDescription: View images running in kubernetes cluster.
```

Comment:
- Kubernetes word is redundant
- Consider removing trailing "."

I'd probably rewrite this as

> Query container images by namespace/cluster.

Reply:
updated short description